libs/api/domains/national-registry/src/lib/shared/models/banMarking.model.ts

```diff
@@ -1,6 +1,8 @@
 import { Field, ObjectType } from '@nestjs/graphql'
 
-@ObjectType()
+//DEPRECATE - This is exceptionFromDirectMarketing field
```

Comment:
Attach to field?

Reply:
This is deprecated as a field in the types using said type

Reply:
Might keep the comment as a cleanup reminder later on